freqtrade/plugins/pairlist/AgeFilter.py

```diff
@@ -86,14 +98,17 @@ def _validate_pair_loc(self, pair: str, daily_candles: Optional[DataFrame]) -> b
             return True
 
         if daily_candles is not None:
-            if len(daily_candles) >= self._min_days_listed:
+            if len(daily_candles) >= self._min_days_listed and \
+                    len(daily_candles) <= self._max_days_listed:
```

Comment:
You will want to be careful with this - as `max_days_listed` can be None - this check will fail with `TypeError: '<=' not supported between instances of 'int' and 'NoneType'` should the first condition ever evaluate to False.